hpeOneView/resources/security/users.py

```diff
@@ -108,8 +108,148 @@ def get_by_userName(self, userName):
             uri = self.URI + '/role/' + userName
             self._helper.validate_resource_uri(uri)
```

Comment:
```validate_resource_uri``` is handled inside ```do_get``` method. So no need to use it again. Also try to avoid ```do_get```, ```do_put```, ```do_post``` methods and replace them with ```get_by_uri```, ```update```, ```create``` methods